src/js/angular/repositories/controllers.js

```diff
@@ -261,6 +261,10 @@ function LocationsAndRepositoriesCtrl($scope, $modal, toastr, $repositories, Mod
             });
     };
 
+    $scope.restartRepository = function (repositoryId) {
+        $repositories.restartRepository(repositoryId);
```

Comment:
Some feedback should be returned to the user, at minimum that the restart is successfully initialized and optionally if possible notification again when the repo is up.